### DISCUSSION TITLE
如果是测试环境的库也接入到这个平台,要求可以自由的增删查改, 使用这个平台只是作为审计使用,这个权限应该怎么配置

### DISCUSSION THREAD

**Discussion by codech** [hhyo/Archery#2357] [Q&A]
### 重现步骤

如果是测试环境的库也接入到这个平台,要求可以自由的增删查改, 使用这个平台只是作为审计使用,这个权限应该怎么配置

### 预期外的结果

请问这个怎么实现呢

### 版本

最新版本

### 部署方式

Docker

**Comment by LeoQuote** [comment]
平台支持设置无需审批，可以关联若干实例，让这些实例自动通过。

另外还可以部署两个archery，专门用来做这些工作，权限可以放开一些，比如所有人都可以审批
